src/policy/target-code.md

```diff
@@ -0,0 +1,25 @@
+# Reviewing target-specific code
+
+When reviewing target-specific code, depending on the [tier] of the target in
+question, different level of scrutiny is expected from reviewers.
+
+For tier 1 targets, the reviewer should perform a full review of the code.
+Essentially treat the code as *not* platform specific.
+
+For tier 2 and tier 3 targets, the reviewer should confirm that the code:
```

Comment:
Briefly reiterating the platform support policy would help motivate this guidance. Something like:

> For tier 1 targets, the reviewer should perform a full review of the code. Essentially treat the code as *not* platform specific. <ins>Tier 1 code is considered "guaranteed to work", and code review is an important aspect of how we uphold this guarantee.</ins>
>
> <ins>Tier 2 targets are considered "guaranteed to build". When reviewing tier 2 and tier 3 target-specific code,</ins> the reviewer should confirm that the code: